File: python/035_firstDigit.py
def firstDigit(inputString):
    '''
    Find the leftmost digit that occurs in a given string.
    '''
    for i in inputString:
        if i.isdigit():
            return i
    return None
        
print(firstDigit("var_1__Int"))
print(firstDigit("q2q-q"))
print(firstDigit("0ss"))

File: arcade/go/arcade-intro/08_Diving Deeper/035_firstDigit.go
func firstDigit(inputString string) string {
	zeroCharValue := 48
	nineCharValue := 57

	for i := range inputString {
		if int(inputString[i]) >= zeroCharValue && int(inputString[i]) <= nineCharValue {
			return string(inputString[i])
		}
	}

	return "-"
}
